js/src/forum/addProfilePane.ts

```diff
@@ -17,15 +20,15 @@ export default function addProfilePane() {
     },
   };
 
-  extend(UserPage.prototype, 'navItems', function (items) {
-    if (app.forum.attribute('canViewMasquerade') || this.user.canEditMasqueradeProfile()) {
-      const edit = this.user.canEditMasqueradeProfile();
+  extend(UserPage.prototype, 'navItems', function (items: ItemList<Mithril.Children>) {
+    if (app.forum.attribute('canViewMasquerade') || this.user?.canEditMasqueradeProfile()) {
```

Comment:
```suggestion
    if (app.forum.attribute<boolean>('canViewMasquerade') || this.user?.canEditMasqueradeProfile()) {
```
Let's also type the forum attribute